package.json

```
@@ -31,7 +31,10 @@
   "dependencies": {
     "@asyncapi/generator-filters": "^1.1.0",
     "@asyncapi/generator-hooks": "^0.1.0",
-    "puppeteer": "^5.2.1"
+    "@asyncapi/generator-react-sdk": "file:../generator-react-sdk",
+    "react": "^17.0.1",
+    "puppeteer": "^5.2.1",
+    "source-map-support": "^0.5.19"
```

Dependencies are removed.
```suggestion
    "puppeteer": "^5.2.1",
```

---

I will remove it after testing with sdk on npm 😄